.circleci/config.yml

```diff
@@ -233,7 +233,7 @@ jobs:
       - image: circleci/node:10.16.3
     environment:
       FIREBASE_PREFIX: << pipeline.git.branch >>-1
-      USE_VIRTUAL_FUNDING: 'FALSE'
+      REACT_APP_FUNDING_STRATEGY: Direct
```

Comment:
Can you add this var to `integration-test-web3torrent` config? That would make the CI pass. I tried it on master.